Packaging/amiga/SIL Open Font License.txt

```diff
@@ -0,0 +1,46 @@
+Digitized data copyright (c) 2010 Google Corporation
```

Comment:
Perhaps we should move the font to `Packaging/resources`?
It's not Amiga-specific, we could use it on other resource-constrained platforms (e.g. Nintendo 3DS)

Reply:
I think I would prefer doing that when it turns out to be needed.